ckan/requirements.txt

```diff
@@ -4,18 +4,18 @@ Babel==2.10.3
 Beaker==1.11.0
 bleach==5.0.1
 blinker==1.5
-boto3==1.34.120
-botocore==1.34.120
+boto3==1.34.128
+botocore==1.34.128
 certifi==2024.6.2
 cffi==1.16.0
 chardet==5.2.0
 charset-normalizer==3.3.2
 ckan @ git+https://github.com/GSA/ckan.git@7159a872ba740069b768fcd2a43cde81a57ee492
 -e git+https://github.com/ckan/ckanext-archiver.git@cbfadf9fbf10405958fdef9f77a7faedc05aa20b#egg=ckanext_archiver
 ckanext-datagovcatalog==0.1.0
-ckanext-datagovtheme==0.2.22
```

Comment:
I think we want this to be 24: https://pypi.org/project/ckanext-datagovtheme/0.2.24/

Reply:
I'm going to bump it in the other PR